app/models/meal.rb

```diff
@@ -0,0 +1,22 @@
+class Meal < ApplicationRecord
+  ### ASSOCIATIONS ###
+
+  has_one_attached :image
+  has_many :ingredients_meals, class_name: 'IngredientMeal'
```

Comment:
add dependent conditions in associations everywhere applicable

Reply:
Done